plugin/pkg/scheduler/api/v1/register.go

```diff
@@ -26,7 +27,16 @@ import (
 var SchemeGroupVersion = unversioned.GroupVersion{Group: "", Version: "v1"}
 
 func init() {
-	api.Scheme.AddKnownTypes(SchemeGroupVersion,
+	addKnownTypes(api.Scheme)
```

Comment:
Yes, I am absolutely not done fixing this code, it's super inconsistent right now. I want to remove all of these init functions, but I left them in so as not to change behavior yet.